apple/LayoutReanimation/REAScreensHelper.h

```diff
@@ -17,5 +18,10 @@
 + (REAUIView *)getScreenWrapper:(REAUIView *)view;
 + (int)getScreenType:(REAUIView *)screen;
 + (bool)isRNSScreenType:(REAUIView *)screen;
++ (bool)isStackChanged:(REAUIView *)view;
++ (REAUIView *)findTopScreenInChildren:(REAUIView *)screen;
++ (REAUIView *)getActiveTabForTabNavigator:(REAUIView *)tabNavigator;
++ (bool)isChild:(REAUIView *)child OfScreen:(REAUIView *)screen;
```

Comment:
This is a nitpick but I think it makes more sense this way
```suggestion
+ (bool)isView:(REAUIView *)view ChildOfScreen:(REAUIView *)screen;
```

Reply:
I prefer the current way.

Reply:
I prefer the current way.